service/matching/matching_engine.go

```diff
@@ -215,6 +221,9 @@ func (e *matchingEngineImpl) Start() {
 	) {
 		return
 	}
+
+	go e.watchMembership()
+	e.serviceResolver.AddListener(e.listenerKey(), e.membershipChangedCh)
```

Comment:
```suggestion
	_ = e.serviceResolver.AddListener(e.listenerKey(), e.membershipChangedCh)
```

Can we eat the error to silence these linter warnings?